static/src/javascripts-legacy/projects/commercial/modules/dfp/render-advert.js

```diff
@@ -156,6 +164,42 @@ define([
                 }) : Promise.resolve();
             }
 
+            function addFeedbackDropdownToggle() {
+                return isRendered ? fastdom.write(function () {
```

Comment:
I'm not sure I'm reading this right, but isn't `addFeedbackDropdownToggle` going to run no matter if there is actually a toggle or not? (same for `applyFeedbackOnClickListeners` but I suppose in that case the queries will return empty arrays so we're fine there)